### PR TITLE
br: public the parameter --ignore-stats

### LINKED ISSUE #48008: public the parameter `--ignore-stats` for br
## Enhancement
public the parameter `--ignore-stats` for br

### PR DESCRIPTION
**Pull request by Leavrth**
### What problem does this PR solve?

Issue Number: close #48008

Problem Summary:
public the parameter --ignore-stats
### What is changed and how it works?
public the parameter --ignore-stats
### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: pkg/domain/domain.go [hunk 1]
```diff
@@ -2238,7 +2238,7 @@ func (do *Domain) UpdateTableStatsLoop(ctx, initStatsCtx sessionctx.Context) err
 	}
 	do.statsHandle.Store(statsHandle)
 	do.ddl.RegisterStatsHandle(statsHandle)
-	// Negative stats lease indicates that it is in test, it does not need update.
+	// Negative stats lease indicates that it is in test or in br binary mode, it does not need update.
```
Comment (BornChanger) [inline pkg/domain/domain.go:2241]: From where can we see stats is set negative for br binary mode?

Comment (Leavrth) [reply pkg/domain/domain.go:2241]: https://github.com/pingcap/tidb/blob/a51ea16fb7e3b9d434f01f44bc33fb15315f64a6/br/cmd/br/backup.go#L50

**Review by BornChanger — APPROVED** [review]

**Review by 3pointer — APPROVED** [review]

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**